tests/spark/autologging/ml/test_pyspark_ml_autologging.py

```diff
@@ -276,7 +279,29 @@ def test_fit_with_a_list_of_params(dataset_binomial):
             mock_set_tags.assert_not_called()
 
 
-def test_should_log_model(dataset_binomial, dataset_multinomial, dataset_text):
+@pytest.fixture()
```

Comment:
Should we test for the StringIndexer -> IndexToString for multiclass classification use cases?
https://spark.apache.org/docs/latest/ml-features.html#indextostring 
or is that going to complicate things?

Reply:
Good point, working on it.